### PR TITLE
Renaming workflow to program

### PR DESCRIPTION
**Pull request by keyleu**
Renames workflows to programs.

workflow-manager is now program-manager.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Valence Protocol
+# Valence Programs
```
Comment (bekauz) [inline README.md:1]: protocol -> programs as well?

File: contracts/program-registry/src/contract.rs [hunk 2]
```diff
@@ -120,19 +118,19 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
...
+            let program = ProgramResponse {
                 id,
-                workflow_config: config,
+                program_config: config,
             };
-            to_json_binary(&workflow)
+            to_json_binary(&program)
         }
         QueryMsg::GetConfigBackup { id } => {
             let config = WORKFLOWS_BACKUP.may_load(deps.storage, id)?;
```
Comment (bekauz) [inline contracts/program-registry/src/contract.rs:128]: `WORKFLOWS_BACKUP` leftover

File: contracts/program-registry/src/state.rs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+use cosmwasm_std::Binary;
+use cw_storage_plus::{Item, Map};
+
+pub const LAST_ID: Item<u64> = Item::new("id");
+pub const WORKFLOWS: Map<u64, Binary> = Map::new("programs");
+pub const WORKFLOWS_BACKUP: Map<u64, Binary> = Map::new("programs_backups");
```
Comment (bekauz) [inline contracts/program-registry/src/state.rs:6]:
```suggestion
pub const PROGRAMS: Map<u64, Binary> = Map::new("programs");
pub const PROGRAM_BACKUP: Map<u64, Binary> = Map::new("program_backups");
```

guessing these are leftovers?

File: README.md (hunk 1 above)
Comment (keyleu) [reply README.md:1]: Yes, the repo will also be renamed eventually

File: contracts/program-registry/src/state.rs (hunk 3 above)
Comment (keyleu) [reply contracts/program-registry/src/state.rs]: Fixed, missed these

**Review by bekauz — APPROVED** [review]

File: contracts/program-registry/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Program registry
+
+This contract is a registry of all the program we instantiated.
+
+This should allow our manager to "lock" and id and store the full program config in the registry.
```
Comment (Art3miX) [inline contracts/program-registry/README.md:5]:
```suggestion
This contract is a registry of all the programs we instantiated.

This should allow our manager to "lock" an id and store the program config in the registry.

Library specific config are not stored in the registry, but can be queried from the specific library contract.
```

**Review by Art3miX — APPROVED** [review]

**Pull request merged**